server/honeybadger/plugins.py

```diff
@@ -26,18 +24,43 @@ def get_coords_from_google(aps):
         data['lng'] = jsondata['location']['lng']
     return data
 
-def get_coords_from_uniapple(ip):
-    logger.info('Geolocating via Uniapple API.')
-    url = 'http://uniapple.net/geoip/?ip={}'.format(ip)
-    content = urllib2.urlopen(url).read()
-    logger.info('Uniapple API response:\n{}'.format(content))
+def get_coords_from_ipstack(ip):
+    logger.info('Geolocating via Ipstack API.')
+    url = 'http://api.ipstack.com/{0}?access_key={1}'.format(ip, app.config['IPSTACK_API_KEY'])
+    content = requests.get(url).content
+    logger.info('Ipstack API response:\n{}'.format(content))
     jsondata = None
     try:
         jsondata = json.loads(content)
     except ValueError as e:
         logger.error('{}.'.format(e))
+
+    # Avoid the KeyError. For some reason, a successful API call to Ipstack doesn't include
```

Comment:
Just wanted to comment on the poor design here by upstack. That flag should always be there.

Reply:
I agree. Makes the error catching more difficult.